articles/references/concepts/stereotypes.md

```diff
@@ -0,0 +1,76 @@
+# Stereotypes
+
+## What is a Stereotype?
+
+>Stereotype is a profile class which defines how an existing metaclass may be extended as part of a profile. It enables the use of a platform or domain specific terminology or notation in place of, or in addition to, the ones used for the extended metaclass.
+
+_from: [uml-diagrams.org](https://www.uml-diagrams.org/stereotype.html)_
+
+According to the quote from the UML Diagrams website, a stereotype defines in UML terms that a given Class will have additional meta-data (in a simplified way).
+
+Intent Architect has adopted this notation called `Stereotype` in order to apply additional meta-data on-top of existing Elements.
+
+## What does Stereotypes look like?
+
+Stereotypes are quite flexible in how they appear. You can have a visible indicator on the Element that you work on or it might be only a field where you need to supply additional information in the form of a text field or the like.
```

Comment:
```suggestion
Stereotypes in Intent Architect are intentionally able to be visually obvious so that users can quickly visually see that a particular element is configured in a particular way without having to dig into an element's details.

Intent Architect offers a lot of flexibility in how stereotypes can appear. They can use one of built in icons, or alternatively any custom image. Additionally, they have customizable display text, this text also supports JavaScript syntax allowing features such as inclusion of stereotype values or conditional display or text content.
```